libs/components/src/lib/option/option.ts

```diff
@@ -58,16 +58,18 @@ export class ListboxOption extends HostSemantics(
 	/**
 	 * The defaultSelected state of the option.
 	 * @public
+	 * @remarks
+	 * HTML Attribute: selected
 	 */
-	@observable
+	@attr({ attribute: 'selected', mode: 'boolean' })
 	defaultSelected = false;
 	protected defaultSelectedChanged(): void {
 		if (!this.dirtySelected) {
 			this.selected = this.defaultSelected;
-
-			if (this.proxy instanceof HTMLOptionElement) {
-				this.proxy.selected = this.defaultSelected;
-			}
+		}
+		if (this.proxy instanceof HTMLOptionElement) {
+			this.proxy.selected = this.defaultSelected;
```

Comment:
But now it sets `this.proxy.selected` even when `dirtySelected`
So I think it needs:
```suggestion
			if(!this.dirtySelected) {
				this.proxy.selected = this.defaultSelected;
			}
```